cclib/parser/psi4parser.py

```diff
@@ -44,6 +45,11 @@ def before_parsing(self):
         # with =>/<= rather than ==>/<==.
         self.subsection = None
 
+        # Regular expression for matching the start of an atomic charges
+        # section; the second format of the header (with square brackets) is
+        # for old Psi4 versions.
+        self.re_atomic_charges_header = re.compile(r"^\s*(\w+) Charges(:?: \(a\.u\.\)| \[a\.u\.\]:)")

```

Comment:
Do you think it's better to spell out which ones we expect here, as in `(Mulliken|Lowdin)` ?

Reply:
According to https://psicode.org/psi4manual/master/oeprop.html there is only one other type (MBIS), so it is not bad to do `(Mulliken|Lowdin|MBIS)`.